docs/sources/logql/template_functions.md

```diff
@@ -371,3 +371,227 @@ Examples:
 {{ if .err hasSuffix "Timeout" }} timeout {{end}}
 {{ if hasPrefix "he" "hello" }} yes {{end}}
 ```
+
+## add 
+
+> **Note:** Added in Loki 2.2.
```

Comment:
Not sure if this should be `2.2`. We may have to change.

Reply:
Yeah this is going to be 2.3